Extension/src/main.ts

```diff
@@ -371,15 +190,15 @@ function handleError(installBlob: InstallBlob, error: any): void {
         }
 
         if (packageError.errorCode) {
-            installBlob.telemetryProperties['error.errorCode'] = removePotentialPII(packageError.errorCode);
+            installBlob.telemetryProperties['error.errorCode'] = util.removePotentialPII(packageError.errorCode);
         }
     } else {
         errorMessage = error.toString();
-        installBlob.telemetryProperties['error.toString'] = removePotentialPII(errorMessage);
+        installBlob.telemetryProperties['error.toString'] = util.removePotentialPII(errorMessage);
     }
 
     // Show the actual message and not the sanitized one
```

Comment:
>// Show the actual message and not the sanitized one [](start = 4, length = 52)

this comment seems wrong

Reply:
I have no idea what was supposed to go there.

Reply:
is it the channel.appendline stuff with the errorMessage? i feel like it is that. i don't know why we are addin ga line here. 

---
In reply to: [160840925](https://github.com/Microsoft/vscode-cpptools/pull/1428#discussion_r160840925) [](ancestors = 160840925)